src/core.ts

```diff
@@ -102,17 +104,27 @@ export const defaults: Options = {
 }
 const isWin = process.platform == 'win32'
 try {
+  setupBash()
+} catch (err) {}
+
+export function setupPowerShell() {
+  if (!isWin) throw new Error('PowerShell is only available on Windows')
```

Comment:
This is not true. PowerShell is also available on macOS and Linux.

Reply:
ok